tools/stconfig/commands.go

```diff
@@ -27,18 +28,36 @@ func getFilePathsByDir(dirName string) ([]string, error) {
 
 // GenKeys generates ED25519 keypair and stores it on the harddrive
 func GenKeys() error {
+	if _, err := os.Stat(*genkeysPrivateKeyFile); os.IsNotExist(err) {
+		return fmt.Errorf("Private key file does not exist: %v", err)
+	}
+	if _, err := os.Stat(*genkeysPublicKeyFile); os.IsNotExist(err) {
+		return fmt.Errorf("Public key file does not exist: %v", err)
+	}
 	return crypto.GeneratED25519Key([]byte(*genkeysPassphrase), *genkeysPrivateKeyFile, *genkeysPublicKeyFile)
 }
 
 // PackBootConfiguration packages a boot configuration containing different
 // binaries and a manifest. The files to be included are taken from the
 // path specified in the provided manifest.json
 func PackBootConfiguration() error {
+	if _, err := os.Stat(*createManifest); os.IsNotExist(err) {
+		return fmt.Errorf("manifest.json does not exist: %v", err)
+	}
 	return bootconfig.ToZip(*createOutputFilename, *createManifest)
 }
 
 // AddSignatureToBootConfiguration TODO:
 func AddSignatureToBootConfiguration() error {
+	if _, err := os.Stat(*signInputBootfile); os.IsNotExist(err) {
+		return fmt.Errorf("Input boot file not found: %v", err)
+	}
+	if _, err := os.Stat(*signPrivKeyFile); os.IsNotExist(err) {
+		return fmt.Errorf("Private key file found: %v", err)
+	}
+	if _, err := os.Stat(*signCertFile); os.IsNotExist(err) {
+		return fmt.Errorf("Certificate file not found: %v", err)
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Here and everywhere above